### ISSUE TITLE
Very easy to bypass

### ISSUE THREAD

**Issue by doctor8296** [AEPKILL/devtools-detector#76]
```js
console.log = () => {};
console.table = () => {};
Function.prototype.constructor = () => {};
setInterval = () => {};

for (let i=0;i <1e6; i++) {
    clearInterval(i);
}

delete Navigatorprototype.userAgent
```

And a lot more.

I am uncertain about the purpose of this library. The only individuals likely to open the developer tools intentionally are those who  exactly knows what they are doing, making this method largely ineffective, even if it technically functions as intended.

**Comment by doctor8296** [comment]
By the way, here another way to stuck the browser without ability to debugg it during crash trigger:
```js
(async function f() {f(), f()})();
```

**Comment by AEPKILL** [comment]
Yes, this library merely adds a little bit of difficulty for the debuggers. If there's a higher demand for anti-debugging, you can try WebAssembly or JVM, or implement the core logic on the server side.

**Comment by doctor8296** [comment]
@AEPKILL, my focus is more on strengthening the system—using non-overridable methods and similar techniques. I’m researching this extensively and working to counter script-based attacks, which are the most common. I’ve developed anti-cheat measures for several browser games, and while I haven’t yet implemented my strongest concept, it's beyond any existing protections out there.

Maybe I will show later some of my tricks and how they can be used to protect this library from almost any "script based attacks"

**Comment by AEPKILL** [comment]
@doctor8296 Your ideas are very interesting. I'm particularly intrigued by the unredefinable methods you mentioned. Can you share some more details about how they work?

**Comment by doctor8296** [comment]
@AEPKILL sure, but I'd prefer to talk privately about it, because some of protection methods us not public yet

**Comment by AEPKILL** [comment]
@doctor8296   Sure, I understand your need for privacy. I think email would be a great way to have this discussion. Please send your thoughts and details to my email address at [a@aepkill.com]. Looking forward to hearing from you soon!

**Comment by AEPKILL** [comment]
@doctor8296  It's almost 11 pm local time. I'm going to bed now. Sorry I might not reply to your emails and messages promptly.

**Comment by nabilfreeman** [comment]
There is always the solution of overriding the loaded source file, changing all isOpen calls to false and refreshing the page which completely negates this library. But I think this is still a decent first line of defense against most normies.

**Comment by doctor8296** [comment]
> There is always the solution of overriding the loaded source file, changing all isOpen calls to false and refreshing the page which completely negates this library. But I think this is still a decent first line of defense against most normies.

@nabilfreeman It is not always. Try remove it from a properly obfuscated source code.
